core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/MultiField.java

```diff
@@ -46,4 +49,21 @@
      * @return Reference to a class
      */
     Class<?> value() default _Default.class;
+
+    /**
+     * Maps to the {@code deleteHint} attribute of this Granite UI component's node.
+     * If set to true, generate the <a href="https://sling.apache.org/documentation/bundles/manipulating-content-the-slingpostservlet-servlets-post.html#delete">
+     * {@code @Delete}</a> attribute for the http request
+     * @return True or false
+     */
+    @PropertyRendering(ignoreValues = "false")
+    boolean deleteHint() default false;
+
+    /**
+     * Maps to the {@code typeHint} attribute of this Granite UI component's node.
+     * Used to specify value of <a href="https://sling.apache.org/documentation/bundles/manipulating-content-the-slingpostservlet-servlets-post.html#typehint">
```

Comment:
(cosmetic) **the** value, and then, wrap `@TypeHint` in `{@code }` the way you did for @Delete above